main/utils.py

```diff
@@ -27,6 +29,15 @@ def webpack_dev_server_url(request):
     return 'http://{}:{}'.format(webpack_dev_server_host(request), settings.WEBPACK_DEV_SERVER_PORT)
 
 
+def now_in_utc():
```

Comment:
Feel free to keep, but we can get the same functionality from https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.timezone.now